server/collaboration/src/utils/ydoc.ts

```diff
@@ -43,3 +51,30 @@ export function yDocCopyXmlField (ydoc: YDoc, source: string, target: string): v
     dstField.insert(0, srcField.toArray().map((item) => (item instanceof YAbstractType ? item.clone() : item)) as any)
   })
 }
+
+/**
+ * @public
+ * Replaces standard clone method that only copies string attributes
+ * https://github.com/yjs/yjs/blob/main/src/types/YXmlElement.js#L97
+ * @param src YXmlElement
+ * @returns YXmlElement
+ */
+export function clone (src: YXmlElement): YXmlElement {
```

Comment:
please add a test when if have a chance, there is a test suite for ydoc utils

Reply:
Done